models/Room.js

```diff
@@ -0,0 +1,17 @@
+import mongoose from 'mongoose';
+
+const RoomSchema = new mongoose.Schema({
+  name: { type: String, required: true },
+  coordinates: {
+    longitude: { type: Number, required: true },
```

Comment:
One day we could probably find something better than long/lat, but for now I don't have a better solution either! Good work